### ISSUE TITLE
[BUG] Vertical lines are't working whith the Minimal Theme.

### ISSUE THREAD

**Issue by So8etnik** [vslinko/obsidian-outliner#406]
**Describe the bug**
Vertical lines are't working whith the Minimal Theme.

**To Reproduce**
Wen i start the Minimal Theme vertical lines immidiatly stop working (i can't clicking on them).

**Desktop (please complete the following information):**
 - Windows 11
 - Obsidian Version: 1.1.9
 - Minimal Theme 6.2.0

**Comment by atnoir** [comment]
same issue. do you find any theme that is compatible with this vertical line feature? for me, it only works in the default theme, but those lines are all misaligned. I tried several themes such as topaz and things, but none of them worked (the lines are just not clickable). thanks.

**Comment by vslinko** [comment]
You are correct, only the default theme is supported. Updated the disclaimer in the README.
